lua/core/screen.lua

```diff
@@ -208,9 +208,12 @@ Screen.text_center_rotate = function(x, y, str, degrees) _norns.screen_text_cent
 -- completes asynchronousely via system callback
 -- (FIXME: see screen.handle_text_extents() or whatever)
 -- @tparam string str : text to calculate width of
-Screen.text_extents = function(str)
-   _norns.screen_text_extents(str)
-end
+Screen.text_extents = function(str) _norns.screen_text_extents(str) end
+
+-- get the current drawing position in the screen surface
+-- @treturn number x
+-- @treturn number y
+Screen.current_point() return _norns.screen_current_point() end
```

Comment:
```
- Screen.current_point() return _norns.screen_current_point() end 
+ Screen.current_point = function() return _norns.screen_current_point() end
```

Above change needs to be made actually. Otherwise there is an error in startup.

```
Oct 25 03:03:44 norns ws-wrapper[2490]: lua: error loading module 'core/screen' from file '/home/we/norns/lua/core/screen.lua':
Oct 25 03:03:44 norns ws-wrapper[2490]:         /home/we/norns/lua/core/screen.lua:216: <eof> expected near 'end'
```

Reply:
huh, no idea how i managed that. sorry!

pushed the change